### PR TITLE
DSY-1325 improving the build 

### PR DESCRIPTION
**Pull request by DobroTora**

### CODE REVIEW

**Review by Parsium — COMMENTED** [review]
Hi @DobroTora 👋🏼 Everything seems fine to me 👍🏼  I've checked the build process and some of the other package commands and can't see any problems related to these packages.

I have a question though regards to the new `tokensExtended.json` file. Could you provide some context behind its addition?

**Comment by DobroTora** [comment]
> Hi @DobroTora 👋🏼 Everything seems fine to me 👍🏼 I've checked the build process and some of the other package commands and can't see any problems related to these packages.
> 
> I have a question though regards to the new `tokensExtended.json` file. Please could you provide some context behind its addition?

Hello @Parsium - extendedTokens.json are a leftover from Table work - I will revert this file as its not related to the current work. Thank you for asking and reviewing :)

File: package.json [hunk 1]
```diff
@@ -37,7 +37,7 @@
     "@semantic-release/release-notes-generator": "^10.0.3",
     "@types/glob": "^7.1.4",
     "dotenv": "^16.0.0",
-    "eslint": "^7.23.0",
+    "eslint": "^7.25.0",
```
Comment (nineteen88) [inline package.json:40]: Out of curiosity - why do we only want to upgrade this two minor releases? The Caret should take care of this already. Granted it's only eslint, so not the be all and end all just wondered if it made sense to update the major version?

**Comment by nineteen88** [comment]
Following the comments I've made I also noticed we have now introduced a package-lock file, when originally the project seems to have been using `yarn`. I'm not sure why we're using yarn in this instance, but I would be tempted to continue with that rather than changing it to `npm` unless anyone disagrees with me?

To resolve the original issue, it shouldn't require us to update the versions in the package.json file but instead you should just be able to run `yarn upgrade` and that should update the lock file accordingly to the latest Minor release of the packages - as the Carets in package.json enable.

**Review by nineteen88 — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -59,6 +59,6 @@
     "scss": "^0.2.4",
     "style-dictionary": "^3.0.1",
     "tinycolor2": "^1.4.2",
-    "typescript": "^4.2.3"
+    "typescript": "^4.6.2"
```
[the remark this answers is not in the document]
Comment (nineteen88) [reply package.json:62]: Decided to just leave change as it is.

File: package.json (hunk 1 above)
Comment (nineteen88) [reply package.json:40]: Decided to just leave change as it is.

**Comment by nineteen88** [comment]
> Following the comments I've made I also noticed we have now introduced a package-lock file, when originally the project seems to have been using `yarn`. I'm not sure why we're using yarn in this instance, but I would be tempted to continue with that rather than changing it to `npm` unless anyone disagrees with me?
> 
> To resolve the original issue, it shouldn't require us to update the versions in the package.json file but instead you should just be able to run `yarn upgrade` and that should update the lock file accordingly to the latest Minor release of the packages - as the Carets in package.json enable.

Following discussions with @edleeks87 - decided we should just leave it as is for now and we will address this later. Ideally we don't want two different lock files, but this has already been pushed to master so we will fix it another time.

**Review by Parsium — APPROVED** [review]

**Pull request merged**